libcontainer/cgroups/fs/cpu.go

```diff
@@ -89,6 +89,25 @@ func (s *CpuGroup) Set(path string, r *configs.Resources) error {
 			period = ""
 		}
 	}
+
+	var burst string
+	if r.CpuBurst != 0 {
```

Comment:
the spec (in https://github.com/opencontainers/runtime-spec/pull/1120) says

>    A value of `0` for indicates that the cgroup can not accumulate any unused bandwidth. It makes the traditional bandwidth control behavior unchanged.

but since we ignore `0` here, this means we can't disable burst during `runc update`.

Reply:
Good catch, thanks! Yes, I just realized after submitting this so I updated the spec accordingly to elaborate.